opentelemetry/CHANGELOG.md

```diff
@@ -10,8 +10,9 @@ of entries, so moving from `IndexMap` to `HashMap` offers slight performance
 gains, and avoids `IndexMap` dependency. This affects `body` and `attributes` of
 `LogRecord`.
 [#1353](https://github.com/open-telemetry/opentelemetry-rust/pull/1353)
-
 - Add `TextMapCompositePropagator` [#1373](https://github.com/open-telemetry/opentelemetry-rust/pull/1373)
+- Remove `global::shutdown_meter_provider`, use `SdkMeterProvider::shutdown`
```

Comment:
Nit: should this go under Removed section?

Reply:
Yeah true